File: app/src/main/assets/templates/Android Game/java/app/src/main/java/$packagename/AndroidLauncher.java
package $packagename;

import android.os.Bundle;

import com.badlogic.gdx.backends.android.AndroidApplication;
import com.badlogic.gdx.backends.android.AndroidApplicationConfiguration;

/* Launches the Android application. */
public class AndroidLauncher extends AndroidApplication {
	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		/* You can adjust this configuration to fit your needs */
		AndroidApplicationConfiguration configuration = new AndroidApplicationConfiguration();
		configuration.useImmersiveMode = true;
		initialize(new GameSuperClass(), configuration);
	}
}

File: app/src/main/assets/templates/Android Game/kotlin/app/src/main/java/$packagename/AndroidLauncher.kt
package $packagename

import android.os.Bundle
import com.badlogic.gdx.backends.android.AndroidApplication
import com.badlogic.gdx.backends.android.AndroidApplicationConfiguration

/* Launches the Android application. */
class AndroidLauncher : AndroidApplication() {
	override fun onCreate(savedInstanceState: Bundle?) {
		super.onCreate(savedInstanceState)
		/* You can adjust this configuration to fit your needs */
		val configuration = AndroidApplicationConfiguration()
		configuration.useImmersiveMode = true
		initialize(GameSuperClass(), configuration)
	}
}